snuba/consumers/rust_processor.py

```diff
@@ -57,6 +57,12 @@ def initialize_processor(
 initialize_processor()
 
 
+def ensure_utc(value: Optional[datetime]) -> Optional[datetime]:
+    if value and value.tzinfo is None:
+        value.replace(tzinfo=timezone.utc)
```

Comment:
this returns a new datetime that you need to return
```suggestion
        return value.replace(tzinfo=timezone.utc)
```

Reply:
ugg, thanks